### DISCUSSION TITLE
这个支持特殊的魔改内核吗

### DISCUSSION THREAD

**Discussion by ghost** [bmax121/APatch#404] [Q&A]
机型Vivo Y52s，底包funtouchos10.5，当前系统LineageOS19_BVN_GSI系统（Android12)，已经完全删除AVB2.0和强制加密，未刷TWRP，CPU是MT6853，已解锁BL，不支持Fastboot但可以用MTKClient刷写任何分区。

内核版本4.14.141+，非GKI

内核有反root组件，使用Termux执行任意目录/su均会返回Operation not permitted，原版Magisk刷入无限重启，目前使用suu面具

但CONFIG_KALLSYMS=y

```
u0_a135@localhost~$ uname -a                         Linux localhost 4.14.141+ #1 SMP PREEMPT Thu Nov 30 17:24:16 CST 2023 aarch64 Android
u0_a135@localhost~$ /system/bin/su --version         bash: /system/bin/su: Operation not permitted
u0_a135@localhost~$ /sbin/suu -c /system/bin/zcat /proc/config.gz | grep -w CONFIG_KALLSYMS
CONFIG_KALLSYMS=y
u0_a135@localhost~$
u0_a135@localhost~$
```

**Comment by momo54181** [comment]
vivo y97也是 4.14.141，CONFIG_KALLSYMS=y
